### DISCUSSION TITLE
Invalid expression format 'FieldExpression'

### DISCUSSION THREAD

**Discussion by zraffaele** [alirezanet/Gridify#79] [General]
Hi,
i add a customoperator, the sample class FreeTextOperator, i register it in the startup class and when in the Filter querystring add this operator for example 'Filter=name#=raff' i have this error
Thanks

**Comment by alirezanet** [comment]
Hi @zraffaele,
Can you provide a bit more details? or it would be great if you can add a reproducible example.
`name#=sometext` according to FreeTextOperator in the docs should work fine.

**Comment by zraffaele** [comment]
I prepare a reproducible project, but i need a couple of days
